traffic_portal/app/src/common/modules/form/capability/form.capability.tpl.html

```diff
@@ -18,41 +18,31 @@
 -->
 
 <div class="x_panel">
-    <div class="x_title">
-        <ol class="breadcrumb pull-left">
-            <li><a ng-click="navigateToPath('/capabilities')">Capabilities</a></li>
-            <li class="active">{{capName}}</li>
-        </ol>
-        <div class="pull-right" role="group" ng-show="!settings.isNew">
-            <button class="btn btn-primary" title="View Endpoints" ng-click="viewEndpoints()">View Endpoints</button>
-            <!--<button class="btn btn-primary" title="View Users" ng-click="viewUsers()">View Users</button>-->
-        </div>
-        <div class="clearfix"></div>
-    </div>
-    <div class="x_content">
-        <br>
-        <form name="capForm" class="form-horizontal form-label-left" novalidate>
-            <div class="form-group" ng-class="{'has-error': hasError(capForm.name), 'has-feedback': hasError(capForm.name)}">
-                <label class="control-label col-md-2 col-sm-2 col-xs-12">Name *</label>
-                <div class="col-md-10 col-sm-10 col-xs-12">
-                    <input name="name" type="text" class="form-control" placeholder="Example: foo-read or foo-write" ng-disabled="!settings.isNew" ng-model="capability.name" ng-pattern="/^\S*$/" required autofocus>
-                    <small class="input-error" ng-show="hasPropertyError(capForm.name, 'required')">Required</small>
-                    <small class="input-error" ng-show="hasPropertyError(capForm.name, 'pattern')">No spaces</small>
-                    <span ng-show="hasError(capForm.name)" class="form-control-feedback"><i class="fa fa-times"></i></span>
-                </div>
-            </div>
-            <div class="form-group" ng-class="{'has-error': hasError(capForm.description), 'has-feedback': hasError(capForm.description)}">
-                <label class="control-label col-md-2 col-sm-2 col-xs-12">Description *</label>
-                <div class="col-md-10 col-sm-10 col-xs-12">
-                    <textarea name="description" type="text" class="form-control" ng-model="capability.description" rows="10" required autofocus></textarea>
-                    <small class="input-error" ng-show="hasPropertyError(capForm.description, 'required')">Required</small>
-                    <span ng-show="hasError(capForm.description)" class="form-control-feedback"><i class="fa fa-times"></i></span>
-                </div>
-            </div>
-            <div class="modal-footer">
-                <button type="button" class="btn btn-danger" ng-show="!settings.isNew" ng-click="confirmDelete(capability)">Delete</button>
-                <button type="button" class="btn btn-success" ng-disabled="capForm.$pristine || capForm.$invalid" ng-click="confirmSave(capability)">{{settings.saveLabel}}</button>
-            </div>
-        </form>
-    </div>
+	<div class="x_title">
```

Comment:
shouldn't this form be gone? or are you still allowing them to "view the capability"? seems like viewing all capabilities in the table would be sufficient.

Reply:
I guess the table is probably sufficient, but yeah, that was the intention here.